src/fprime_gds/common/communication/ccsds/space_data_link.py

```diff
@@ -30,12 +32,31 @@ class SpaceDataLinkFramerDeframer(FramerDeframer):
     )
     CRC_CALCULATOR = crc.Calculator(CRC_CCITT_CONFIG)
 
+    # For backwards compatibility if not found in dictionary (loaded by ConfigManager)
+    FALLBACK_SCID = 0x44
+    FALLBACK_FRAME_SIZE = 1024
+
+
     def __init__(self, scid, vcid, frame_size):
```

Comment:
Frame size, SCID, and VCID seem like they should only source from the dictionary, no?  These constants must be in-line with FSW.